ovos_bus_client/session.py

```diff
@@ -508,14 +518,13 @@ def sync(cls, message=None):
     @classmethod
     def connect_to_bus(cls, bus):
         cls.bus = bus
-        cls.bus.on("ovos.session.sync",
-                   cls.handle_default_session_request)
+        cls.bus.on("recognizer_loop:record_begin", cls.handle_recording_start)
+        cls.bus.on("recognizer_loop:record_end", cls.handle_recording_end)
+        cls.bus.on("recognizer_loop:audio_output_start", cls.handle_audio_output_start)
+        cls.bus.on("recognizer_loop:audio_output_end", cls.handle_audio_output_end)
+        cls.bus.on("ovos.session.sync", cls.handle_default_session_request)
         cls.sync()
 
-    @classmethod
```

Comment:
What's the reason for cutting this method?

Reply:
just moved to the bottom so all event handlers are grouped together